libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -198,6 +199,12 @@ void AP_Proximity::init()
             num_instances = instance+1;
         break;
 #endif
+#if AP_PROXIMITY_SCRIPTING_ENABLED
+        case Type::Lua_Scripting:
```

Comment:
nitpick: normally we just call it Scripting rather than Lua_Scripting.  Not a blocker of course.